Example/TABTestKit_ExampleUITests/PageTests.swift

```diff
@@ -17,21 +17,17 @@ final class PageTests: TABTestCase {
         guard #available(iOS 13, *) else {
             return
         }
+        
         Scenario("Navigate to page view") {
             Given(I: complete(biometricLoginScreen))
             And(I: see(tabBarScreen))
             When(I: tap(tabBarScreen.pageTabBarButton))
             Then(I: see(pageScreen))
         }
         
-        // TODO: Replace with 'scroll until value of X is Y' as part of https://github.com/theappbusiness/TABTestKit/issues/72
         Scenario("Scroll page view") {
             Given(I: see(pageScreen))
-            When(I: pageScreen.scrollView.scroll(.right))
-            And(I: pageScreen.scrollView.scroll(.right))
-            And(I: pageScreen.scrollView.scroll(.right))
-            And(I: pageScreen.scrollView.scroll(.right))
-            And(I: pageScreen.scrollView.scroll(.right))
+            When(I: scroll(pageScreen.scrollView, .right, until: pageScreen.pageIndicator, valueIs: "page 6 of 6"))
```

Comment:
I don't know what the other could think but will be great instead to have a hardcoded string value for `valueIs`. We could use a struct as the last parameter so we could reduce misspelling for example.
```swift
struct Page {
  var currentPage: UInt
  var numberOfPages: UInt
  var value: String {
    "page \(currentPage) of \(numberOfPages)"
  }
}

Reply:
Thanks Roger, I will wait for more comments, I like above implementation but I only need these two variables with one particular value.